README.md

```diff
@@ -124,6 +148,14 @@ Because of their generality, vouchers can be used in a wide range of application
 | ERC-1155 | Token contract | `safeTransferFrom(address,address,uint256,uint256,data)` [:page_facing_up:](https://eips.ethereum.org/EIPS/eip-1155#specification) |
 | ERC-1155 | Token contract | `safeBatchTransferFrom(address,address,uint256[],uint256[],data)` [:page_facing_up:](https://eips.ethereum.org/EIPS/eip-1155#specification) [^3] |
 
+Please note that the voucher payload should be encoded according to the [Ethereum ABI specification for calling contract functions](https://docs.soliditylang.org/en/v0.8.19/abi-spec.html). As such, it should start with the first four bytes of the Keccak-256 hash of the function selector string (as given in the table above), followed by the ABI-encoded parameter values.
```

Comment:
The function **selector** is the first four bytes of the Keccak-256 hash of the function **signature**. For example, the EIP-20 transfer function has signature `transfer(address,uint256)` and selector `0xa9059cbb`.

Reply:
Done in https://github.com/cartesi/rollups/pull/42/commits/e4ef2ae0655fe2b21af3afedda7828fbdb4aab80